src/Commands/MakeActionCommand.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace KoalaFacade\DiamondConsole\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Filesystem\FileNotFoundException;
+use Illuminate\Filesystem\Filesystem;
+use KoalaFacade\DiamondConsole\Actions\StubResolver\CopyStubAction;
+
+class MakeActionCommand extends Command
+{
+    protected $signature = 'diamond:action {name} {domain} {--m|migration} {--force}';
```

Comment:
what migration purpose? i don't think we need it in Action command

Reply:
I forgot about this, I'll fix now